url/src/lib.rs

```diff
@@ -73,6 +73,9 @@ assert!(data_url.fragment() == Some(""));
 # run().unwrap();
 ```
 
+## Serde
+
+Add `serde` feature to complement `url::Url` with the `Deserialize` and `Serialize`.
```

Comment:
Nit: I would find this clearer as "Enable the `serde` feature to include `Deserialize` and `Serialize` implementations for `url::Url`."

Reply:
Agreed, this is much better, thanks!